Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -34,9 +32,25 @@ public struct Configuration: Equatable {
     public let rules: [Rule]
     public var rootPath: String?              // the root path to search for nested configurations
     public var configurationPath: String?     // if successfully loaded from a path
-    public var hash: Int?
     public let cachePath: String?
 
+    public var cacheDescription: String {
+        let cacheRulesDescriptions: [String: Any] = rules.reduce([:]) { accu, element in
+            var accu = accu
+            accu[type(of: element).description.identifier] = element.configurationDescription
```

Comment:
the `configurationDescription` of `ConditionalReturnsOnNewline` is `"N/A"` for some reason. We probably should change that as well.